.github/workflows/chainlib.yml

```diff
@@ -62,7 +66,7 @@ jobs:
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.target }}
           override: true
-      - run: cargo build --target ${{ matrix.target }} --release
+      - run: cargo build --target ${{ matrix.target }} --release --features=${{ matrix.feature }}
```

Comment:
are the features mutually exclusive ? what about build with `--all-features`?

Reply:
yes, it seems ok to use `--all-features`, I'll use it.